packages/server/src/adapters/fetch/types.ts

```diff
@@ -3,11 +3,12 @@ import { HTTPBaseHandlerOptions } from '../../http/internals/types';
 
 export type FetchCreateContextFnOptions = {
   req: Request;
+  headers: Headers;
```

Comment:
It's not clear from this variable name that this is _response_ headers.

Ideas, we should at a minimum to the last two I think?

- Call it `resHeaders` / `responseHeaders` 
- Add JSDoc that this is **response** headers
- Add info on https://trpc.io/docs/fetch